### DISCUSSION TITLE
.NET Framework 4.8 to .NET 6 migration

### DISCUSSION THREAD

**Discussion by giscus** [linkdotnet/Blog.Discussions#72] [General]
# .NET Framework 4.8 to .NET 6 migration

https://steven-giesel.com/blogPost/f531dded-44af-4341-871c-d54875877900

**Comment by MPrtenjak** [comment]
Hi,

I recently released a GitHub project for a local conference presentation with detailed steps to upgrade an old .NET Framework WinForms app to .NET WinForms and then upgrade with Blazor Hybrid. 

So also what to do after Upgrade Assistant and how Upgrade Assistant is not always right :)

https://github.com/MPrtenjak/WinForms_2_BlazorHybrid

PS: Great article and a picture that I will (also) use on the conference ;)

**Reply by linkdotnet** [reply]
Hey @MPrtenjak,

That is a very nice repository and also a cool thing to show the step-by-step migration as commits!

**Comment by rabbagast** [comment]
Great article, thanks!
I see no reason to upgrade existing netstandard2.0 libraries to net6.0 though, I find it more flexible to keep them so they can (potentially) be reused anywhere?

**Reply by linkdotnet** [reply]
Thanks!
For the time being, it is absolutely fine to use `netstandard` instead of a version with a runtime (`net6.0`, `net7.0`, ...).
Just be aware that Microsoft discontinued the `netstandard` approach. New APIs and Language features might not be available in `netstandard` projects.
